.github/workflows/buf.yaml

```diff
@@ -0,0 +1,48 @@
+name: buf
+on:
+  push:
+    paths:
+      - 'proto/**'
+  workflow_dispatch:
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: bufbuild/buf-setup-action@v1
+        if: success()
+        with:
+          version: 1.5.0
+      - uses: bufbuild/buf-lint-action@v1.0.1
+        if: success()
+        with:
+          buf_token: ${{ secrets.BUF_TOKEN }}
+  breaking:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: bufbuild/buf-setup-action@v1
+        if: success()
+        with:
+          version: 1.5.0
+      - uses: bufbuild/buf-breaking-action@v1.1.0
+        if: success()
+        with:
+          against: 'https://github.com/bufbuild/connect-demo.git#branch=main'
+  push:
+    runs-on: ubuntu-20.04
+    if: github.ref == 'refs/heads/main'
+    needs:
+      - lint
+      - breaking
+    steps:
+      - uses: actions/checkout@v3
+      - uses: bufbuild/buf-setup-action@v1
+        if: success()
+        with:
+          version: 1.5.0
```

Comment:
I wonder why we're using 1.5.0 instead of the latest 1.6.0. But otherwise this looks solid.

Reply:
Good point.  Updated.